pkg/storages/sh/storage.go

```diff
@@ -59,7 +59,11 @@ func NewStorage(config *Config) (*Storage, error) {
 	client := NewSFTPLazy(address, sshConfig)
 
 	path := storage.AddDelimiterToPath(config.RootPath)
-	folder := NewFolder(client, path)
+	var folder storage.Folder = NewFolder(client, path)
```

Comment:
why not to use compact var initialization here?

Reply:
Because `NewFolder` returns `*sh.Folder` struct, and `storage.Folder` is an interface.
It's then passed to `storage.WrapRootFolder` that works with `storage.Folder`.

I can change the declaration to `folder := storage.Folder(NewFolder(client, path))`, but it looks worse IMHO.